.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: scala
+
+# These directories are cached to S3 at the end of the build
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+
+jdk:
+  - oraclejdk8
+
+script:
```

Comment:
Right now it asks for project name and Akka HTTP version, with Akka and Scala it's 4 in total, all with defaults. I guess it's still OK.